codespell_lib/data/dictionary.txt

```diff
@@ -8339,22 +8339,32 @@ ivocation->invocation
 iwithout->without
 iwll->will
 iwth->with
+jagwar->jaguar
+jalusey->jalousie
```

Comment:
While it turns out this is a word ( https://en.wikipedia.org/wiki/Jalousie_window ), the main suggestion should probably be jealousy.

Reply:
Done

Reply:
You could still suggest jalousie if you want?

Reply:
Done